pipelines/nextflow/modules/manifest/manifest_maker.nf

```diff
@@ -14,8 +14,9 @@
 // limitations under the License.
 
 process MANIFEST {
-    tag "Manifest_$meta.accession"
+    tag "manifest_$meta.accession"
     label 'adaptive'
+    label 'cached'
```

Comment:
This should not be cached as well: this is data that we generate and we don't reuse it.